src/illuminatio/k8s_util.py

```diff
@@ -1,15 +1,25 @@
+"""
+File with several useful functions for interacting with k8s
+"""
+
 import kubernetes as k8s
 from illuminatio.host import Host
 from illuminatio.util import CLEANUP_LABEL, validate_cleanup_in, CLEANUP_ON_REQUEST, CLEANUP_ALWAYS, ROLE_LABEL
 
 
 def init_service_account_for_runners(name, namespace):
+    """
+    Creates, initializes and returns a ServiceAccount for the illuminatio runner
+    """
     sa_labels = {CLEANUP_LABEL: CLEANUP_ON_REQUEST, ROLE_LABEL: "runner-service-account"}
     sa_meta = k8s.client.V1ObjectMeta(name=name, namespace=namespace, labels=sa_labels)
     return k8s.client.V1ServiceAccount(metadata=sa_meta)
 
 
 def init_role_binding_for_service_account(namespace, name, sa_name):
+    """
+    Creates, initializes and returns a ClusterRoleBinding for the illuminatio service account
+    """
     # role binding for account, granting it admin rights for the moment (TODO limit rights)
```

Comment:
ToDo can be removed since we have a custom role: https://github.com/inovex/illuminatio/blob/master/src/illuminatio/manifests/cluster-role.yaml ?